### DISCUSSION TITLE
PHISHED_PHISHTANK being cached somewhere?

### DISCUSSION THREAD

**Discussion by ManChicken1911** [rspamd/rspamd#4904] [Q&A] [answered]
I'm getting a Phishtank hit on messages from Reason Studios:

PHISHED_PHISHTANK (10) [hp http://www.phishtank.com/phish_detail.php?phish_id=8453552]

But that phish_id seems to be invalid, the link just redirects to the home page and if I manually browse through the history I can't find that number. 8453551 is there, and 8453553 is, but not 8453552.

Does anyone know if these being cached or something from a different server, or is some database on their end out of sync through whatever API or mechanism rspamd is looking up with? I've seen the same ID over multiple days so it's not a case of "it just got deleted before you looked"

**Comment by fatalbanana** [comment, marked as answer]
It makes DNS queries to `<some hash>.phishtank.rspamd.com` & data there wasn't being updated; it should be better again.

**Reply by ManChicken1911** [reply]
Ahhhh. Thanks.
